packages/components/Molecules/DataTable/DataTable.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <div class="rpl-data-table">
+    <div v-if="caption" class="rpl-data-table__caption">{{ caption }}</div>
+    <!-- Desktop display -->
+    <table v-if="headers || rows" class="rpl-data-table__table">
+      <thead v-if="headers">
+        <tr><th v-for="(header, index) in headers" :key="`header${index}`" v-html="header" /></tr>
+      </thead>
+      <tbody v-if="rows">
+        <tr v-for="(row, index) in rows" :key="`row${index}`">
+          <td v-for="(col, index) in row" :key="`col${index}`" v-html="col" />
+        </tr>
+      </tbody>
+    </table>
+    <!-- Mobile display -->
+    <div class="rpl-data-table__mobile-layout">
+      <div v-for="(item, i) in responsiveItems" class="rpl-data-table__dl-container" :key="`item${i}`">
+        <dl v-for="(header, j) in responsiveHeaders" :key="`header${j}${i}`">
+          <dt :key="`dt${i}${j}`">{{ header }}</dt>
+          <dd :key="`dd${i}${j}`">{{ item[j] }}</dd>
+        </dl>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'RplDataTable',
+  props: {
+    caption: {
+      type: String,
+      default: ''
+    },
+    isRowOriented: {
+      type: Boolean,
+      default: true
+    },
+    items: {
+      type: Object,
```

Comment:
Any reason `items` is a `Object` instead of `Array`?

Reply:
Backend returns this format. I transposed this to array on mounted so easier to manipulate.

Reply:
Ok. understood now. Can you please check my new comments? I think if possible, use mapping filter to convert the CMS data is more easy for others to understand.